src/LtiDeepLink.php

```diff
@@ -31,10 +31,71 @@ public function getResponseJwt(array $resources): string
 
         // https://www.imsglobal.org/spec/lti-dl/v2p0/#deep-linking-request-message
         // 'data' is an optional property which, if it exists, must be returned by the tool
-        if (isset($this->deep_link_settings['data'])) {
-            $message_jwt[LtiConstants::DL_DATA] = $this->deep_link_settings['data'];
+        if (isset($this->settings()['data'])) {
+            $message_jwt[LtiConstants::DL_DATA] = $this->settings()['data'];
```

Comment:
Should there also be a function to publicly expose the `data` attribute?

Reply:
Data (I believe) is some cryptographically random string that just needs to be included in the JWT payload when it's passed back. To my knowledge, there's no reason that an app will need access to this data, since this library handles building the JWT token. If we learn otherwise, then yes, we should definitely build a function for it.